lib/helpstation/fetchers.rb

```diff
@@ -2,6 +2,50 @@ module Helpstation
   module Fetchers
     NotFoundError = Class.new(StandardError)
 
+    # Helps to run fetchers in parallel
+    #
+    # All output from the fetchers are merged and then returned. This means
+    # that if two fetchers return results with the same key then one will
+    # overwrite the other.
+    #
+    # @example
+    #   process ParallelFetcher[
+    #     OperatorFetcher,
+    #     VisitorFetcher
+    #   ], NOT_FOUND_ERROR
+    #
+    class ParallelFetcher < Processor
```

Comment:
This doesn't have anything fetcher specific other than the requirement that the result must be a `Hash`. 
How about ParallelProcessor?